release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3355,6 +3355,44 @@ This section will continue to be updated over time to provide notes on enhanceme
 ====
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
+
+[id="ocp-4-16-11_{context}"]
+=== RHBA-2024:6401 - {product-title} {product-version}.11 bug fix update
+
+Issued: 10 September 2024
+
+{product-title} release {product-version}.11 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:6401[RHBA-2024:6401] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:6404[RHBA-2024:6404] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.11 --pullspecs
+----
+
+[id="ocp-4-16-11-known-issues_{context}"]
+==== Known issues
+
+* {product-rosa} hosted control planes (HCP) and {product-title} clusters will fail to add new nodes in MachinePool versions older than 4.15.23. As a result, some upgrades are blocked. To see what clusters are affected and the recommended workaround, see (link:https://access.redhat.com/articles/7085666[ROSA upgrade issue mitigation for HOSTEDCP-1941 ]). (link:https://issues.redhat.com/browse/OCPBUGS-39447[*OCPBUGS-39447*])
+
+[id="ocp-4-16-11-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the `noProxy` field from the cluster-wide Proxy wasn't taken into account while configuring proxying for the Platform Prometheus remote write endpoints. With this release, Cluster Monitoring Operator (CMO) will no longer configure proxying for any remote write endpoint whose URL should bypass proxy according to `noProxy`. (link:https://issues.redhat.com/browse/OCPBUGS-39170[*OCPBUGS-39170*])
+
+* Previously, Red{nbsp}Hat HyperShift periodic conformance jobs failed because of changes to the core operating system. These failed jobs caused the OpenShift API deployment to fail. With this release, an update recursively copies individual trusted certificate authority (CA) certificates instead of copying a single file, so that the periodic conformance jobs succeed and the OpenShift API runs as expected (link:https://issues.redhat.com/browse/OCPBUGS-38942[*OCPBUGS-38942*])
```

Comment:
```suggestion
* Previously, Red{nbsp}Hat HyperShift periodic conformance jobs failed because of changes to the core operating system. These failed jobs caused the OpenShift API deployment to fail. With this release, an update recursively copies individual trusted certificate authority (CA) certificates instead of copying a single file, so that the periodic conformance jobs succeed and the OpenShift API runs as expected. (link:https://issues.redhat.com/browse/OCPBUGS-38942[*OCPBUGS-38942*])
```